### ISSUE TITLE
Version 2.2.1 and Html5QrcodeScanType undefined

### ISSUE THREAD

**Issue by ROBERT-MCDOWELL** [mebjas/html5-qrcode#494]
**Describe the bug**
Html5QrcodeScanType is not defined despite of the version 2.2.1 added feature

**To Reproduce**
following the exemple provided after imported the minified html5-qrcode script

```
function onScanSuccess(decodedText, decodedResult) {
  // handle the scanned code as you like, for example:
  console.log(`Code matched = ${decodedText}`, decodedResult);
}

let config = {
  fps: 10,
  qrbox: {width: 100, height: 100},
  rememberLastUsedCamera: true,
  // Only support camera scan type.
  supportedScanTypes: [Html5QrcodeScanType.SCAN_TYPE_CAMERA]
};

let html5QrcodeScanner = new Html5QrcodeScanner(
  "reader", config, /* verbose= */ false);
html5QrcodeScanner.render(onScanSuccess);
```

**Expected behavior**
array Html5QrcodeScanType must be set and not only declared

**Comment by panimaya-gembrill** [comment]
@ROBERT-MCDOWELL Any updates on this issue

**Comment by rlueder** [comment]
@panimaya-gembrill I opened #519 to solve the issue, it was a fairly small fix.

**Comment by mebjas** [comment]
Fixed in https://github.com/mebjas/html5-qrcode/pull/521, thanks to @rlueder 

Will be published in v2.2.4
